bindep.txt

```diff
@@ -0,0 +1,2 @@
+python3-lxml [(platform:redhat platform:base-py3)]
+python-lxml [(platform:redhat platform:base-py2)]
```

Comment:
ansible-builder will consult requirements.txt for python depenencies, and bindep.txt for system packages, right?
But the weird `rpm-py-installer` is living in requirements-dev.txt.

Reply:
Correct. `requirements-dev.txt` is not what `ansible-builder` will look at.

Reply:
Who else is looking at `requirements.txt` should we move all the "optional" requirements there?

Reply:
besides `ansible-builder` (for pure python deps) and our CI? nobody.

we could move python-debian there, yeah